runtime/vm/intfunc.c

```diff
@@ -173,6 +173,7 @@ J9InternalVMFunctions J9InternalFunctions = {
 	objectMonitorEnterBlocking,
 	fillJITVTableSlot,
 	findArgInVMArgs,
+	findArgInVMArgsImpl,
```

Comment:
this is not needed either, since `findArgInVMArgsImpl` is only being called from the j9vm lib